.gitignore

```diff
@@ -1,8 +1,10 @@
 .DS_Store
+coverage/*
```

Comment:
Can we retain the coverage report? We can add it to .npmignore to remove it from the package.

Reply:
i thought of not adding bcoz, we dont have a full coverage, if needed kindly add it as part of your open PR